src/Common/src/Interop/Windows/NtDll/Interop.NtQueryInformationProcess.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class NtDll
+    {
+        [DllImport(Libraries.NtDll, CharSet = CharSet.Unicode)]
+        internal static extern int NtQueryInformationProcess(SafeProcessHandle hProcess, PROCESSINFOCLASS pic, ref PROCESS_BASIC_INFORMATION pbi, int cb, ref int pSize);
```

Comment:
Let's align this with the current docs. Here is the recommended way to do this:

https://docs.microsoft.com/en-us/windows/desktop/api/winternl/nf-winternl-ntqueryinformationprocess

``` C#
internal static extern unsafe uint NtQueryInformationProcess(
    SafeProcessHandle ProcessHandle,
    PROCESSINFOCLASS ProcessInformationClass,
    void* ProcessInformation,
    uint ProcessInformationLength,
    out uint ReturnLength);

// Call like this
PROCESS_BASIC_INFORMATION info = default;
uint status = NtQueryInformationProcess(SafeHandle, PROCESSINFOCLASS.ProcessBasicInformation, &info, (uint)sizeof(PROCESS_BASIC_INFORMATION), out _);

// At least assert on debug for the result - if we decide to throw, convert the error to a Win32 error via RtlNtStatusToDosError (see FileSystemEnumerator.Win32.cs for an example of this)
```

Reply:
Thanks much! Done.